src/utils/DataProcessingUtils.test.js

```diff
@@ -112,4 +122,59 @@ describe('DataProcessingUtils', () => {
 
   });
 
+  describe('processEntityValue', () => {
+
+    test('should return mapped value wrapped in array', () => {
+      const mockMapper = jest.fn();
+      const testMapper = (value) => {
+        mockMapper(value);
+        return value;
+      };
+
+      const mockMappers = {
+        [VALUE_MAPPERS]: {
+          test: testMapper
+        }
+      };
+
+      const UNWRAPPED_VALUES = [
+        'OpenLattice',
+        ['OpenLattice'],
+        List(['OpenLattice']),
+        { Open: 'Lattice' },
+        Map({ Open: 'Lattice' }),
+        true,
+        false,
+        0,
+        1,
+      ];
+
+      const WRAPPED_VALUES = [
+        ['OpenLattice'],
+        ['OpenLattice'],
+        List(['OpenLattice']),
+        [{ Open: 'Lattice' }],
+        [Map({ Open: 'Lattice' })],
```

Comment:
`processEntityValue` should be returning plain values as the result is eventually passed off to a request. `processEntityValue` should never return `List(...)` or `[Map(...)]`

Reply:
This was just to show that `processEntityValue` only skips wrapping values in an array if the value is not a List or Array. So if you have a mapper that needs to return an object that describes binary data with a mime type, it is not interfered with.